strongbox-commons/src/main/resources/application.yaml

```diff
@@ -6,6 +6,8 @@ spring:
   servlet:
     multipart:
       enabled: true
+      max-file-size: 1MB
```

Comment:
Why 1 MB?

Reply:
Default value is 1 MB , so in order to have default behaviour kept this as 1MB.

Reply:
Can you check, if this property can be disabled, if you set it to `0`, or `-1`, or does it need an actual value?

Reply:
`-1` will be treated as unlimited.

Reply:
Quick question - can we use a proxy value for this? Maybe `${strongbox.max-file-size}` and `${strongbox.max-request-size}` which are actually configured in `strongbox.yaml`? :) 

I think this value can overwritten if you export an env var `SPRING_SERVLET_MAX_FILE_SIZE || SPRING_SERVLET_MAX_REQUEST_SIZE`.